projects/core/src/process/store/selectors/process.selectors.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { TestBed } from '@angular/core/testing';
+
+import { StoreModule } from '@ngrx/store';
+
+import { PROCESS_FEATURE } from '../process-state';
+import * as fromReducers from '../reducers';
+
+describe('Process selectors', () => {
+  // let store: Store<ProcessState>;
```

Comment:
Dead code?

Reply:
yes, for now. it will be used in the facade ticket

Reply:
Ok.